plugins/nginx-vhosts/commands

```diff
@@ -76,6 +77,7 @@ EOF
       fi
 
       NOSSL_SERVER_NAME=$(echo $NONSSL_VHOSTS | tr '\n' ' ')
+      docker cp $DOKKU_APP_CONTAINER_ID:/app/nginx.conf $DOKKU_ROOT/$APP/nginx.conf.template 2> /dev/null || true
```

Comment:
> Also, does this fail if an nginx.conf.template file already exists?

No, it will copy over a new configuration file on every deploy, that way changes you make and `git push` will update the Nginx in sync with your code.

> What is the output of this during a deploy?

`Overriding default nginx.conf with detected nginx.conf.template`